src/applications/disability-benefits/686c-674-v2/components/SpouseViewField.jsx

```diff
@@ -4,13 +4,26 @@ export default function SpouseViewField({ formData }) {
   const { first, middle, last, suffix } = formData.fullName;
 
   return (
-    <div className="vads-u-padding--2">
-      <strong>
+    <div className="vads-u-display--flex">
+      <h4 className="vads-u-margin-y--2">
         {first} {middle && `${middle} `}
         {last}
         {suffix && `, ${suffix}`}
-      </strong>
+      </h4>
       <br />
     </div>
   );
 }
+
+export const FormerSpouseHeader = ({ formData }) => {
+  const { first, last } = formData.fullName;
+  return (
+    <legend className="schemaform-block-title" id="root__title">
+      <h3 className="vads-u-color--gray-dark vads-u-margin-top--0">
+        Former marriage to {first?.[0].toUpperCase()}
```

Comment:
Thoughts on using lodash capitalize here?

Reply:
That's a great point - will test that out